finance-api/user-guides/fetch-purchase-information-for-transactions.md

```diff
@@ -0,0 +1,168 @@
+Fetch purchase information for transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions. The `originatingTransactionUuid` of a transaction in the Finance API corresponds to the `uuid` of `payments` in the Purchase API.  
+> **Note:** Currently, fetching purchase information is available only for transactions that are made with cards and alternative payment methods (APMs) like PayPal QRC through Zettle.
+
+The Finance API records transactions for payment types that are supported by the Purchase API except `IZETTLE_CASH`, `SWISH`, `VIPPS`, `MOBILE_PAY`, `GIFTCARD`, `STORE_CREDIT`, and `IZETTLE_INVOICE`. 
+> **Note:**  If `IZETTLE_INVOICE` is paid with a card, the purchase will be recorded as `CARD_PAYMENT` in the Finance API.
+
+The following table shows the payment methods, payment types that are supported by the Purchase API, and the corresponding transaction types that are supported by the Finance API. 
```

Comment:
```suggestion
The following table shows the payment methods and types supported by the Purchase API, and the corresponding transaction types supported by the Finance API. 
```

Reply:
Payment types are a bit general and independent of the APIs. So will keep it as is